style-transfer/fast_style_transfer_net.js

```diff
@@ -0,0 +1,234 @@
+
+    const add7 = this.buildSubNetwork_(builder, conv2D6, variableMul6, variableAdd6);
+    const add8 = builder.add(add5, add7); // next input
+    const conv2D7 = builder.conv2d(builder.pad(add8, padding1, {mode: 'reflection'}), weightConv7);
+
+    const add9 = this.buildSubNetwork_(builder, conv2D7, variableMul7, variableAdd7);
+    const relu5 = builder.relu(add9);
+    const conv2D8 = builder.conv2d(builder.pad(relu5, padding1, {mode: 'reflection'}), weightConv8);
+
+    const add10 = this.buildSubNetwork_(builder, conv2D8, variableMul8, variableAdd8);
+    const add11 = builder.add(add8, add10); // next input
+    const conv2D9 = builder.conv2d(builder.pad(add11, padding1, {mode: 'reflection'}), weightConv9);
+
+    const add12 = this.buildSubNetwork_(builder, conv2D9, variableMul9, variableAdd9);
+    const relu6 = builder.relu(add12);
+    const conv2D10 = builder.conv2d(builder.pad(relu6, padding1, {mode: 'reflection'}), weightConv10);
+
+    const add13 = this.buildSubNetwork_(builder, conv2D10, variableMul10, variableAdd10);
+    const add14 = builder.add(add11, add13); // next input
+    const conv2D11 = builder.conv2d(builder.pad(add14, padding1, {mode: 'reflection'}), weightConv11);
+
+    const add15 = this.buildSubNetwork_(builder, conv2D11, variableMul11, variableAdd11);
+    const relu7 = builder.relu(add15);
+    const conv2D12 = builder.conv2d(builder.pad(relu7, padding1, {mode: 'reflection'}), weightConv12);
+
+    const add16 = this.buildSubNetwork_(builder, conv2D12, variableMul12, variableAdd12);
+    const add17 = builder.add(add14, add16);
+    const convTranspose0 = builder.conv2d(add17, weightConvTranspose0,
+        {transpose: true, strides: [2, 2], outputSizes: [270, 270]});
+
+    const add18 = this.buildSubNetwork_(builder, convTranspose0, variableMul13, variableAdd13);
+    const relu8 = builder.relu(add18);
+    const convTranspose1 = builder.conv2d(relu8, weightConvTranspose1,
+        {transpose: true, strides: [2, 2], outputSizes: [540, 540]});
+
+    const add19 = this.buildSubNetwork_(builder, convTranspose1, variableMul14, variableAdd14);
+    const relu9 = builder.relu(add19);
+    const conv2D13 = builder.conv2d(builder.pad(relu9, padding4, {mode: 'reflection'}), weightConv13);
+
+    const add20 = this.buildSubNetwork_(builder, conv2D13, variableMul15, variableAdd15);
+    const output = builder.add(builder.mul(builder.tanh(add20), constMul0), constAdd0);
+    this.model_ = builder.createModel({'output': output});
+
+    this.loadTime_ = (performance.now() - start).toFixed(2);
+    console.log(`  done in ${this.loadTime_} ms.`);
+  }
+
+  async compile(options) {
+    console.log('- Compiling... ');
+    const start = performance.now();
+    this.compiledModel_ = await this.model_.compile(options);
+    this.compileTime_ = (performance.now() - start).toFixed(2);
+    console.log(`  done in ${this.compileTime_} ms.`);
```

Comment:
I'd suggest to move the performance logging code to where the usage of this method.